tst/standard/isomorph.tst

```diff
@@ -256,7 +256,7 @@ gap> A := [DigraphFromDiSparse6String(".[{?`abcdefghijklmnopqrstuvwxyz"),
 [ <immutable digraph with 28 vertices, 28 edges>, 
   <immutable digraph with 28 vertices, 28 edges>, 
   <immutable digraph with 28 vertices, 28 edges>, 
-  <immutable digraph with 28 vertices, 48 edges> ]
+  <immutable symmetric digraph with 28 vertices, 48 edges> ]
```

Comment:
This test has nothing to do with the Tietze feature; it seems the newest version of Digraphs automatically assigns the `IsSymmetricDigraph` attribute to digraphs loaded using `DigraphFromSparse6String`. This test should be updated regardless of the status of the Tietze PR.